openjpa-jdbc/src/main/java/org/apache/openjpa/jdbc/schema/Constraint.java

```diff
@@ -250,31 +241,48 @@ public String toString() {
         return "<" + name.toLowerCase() + ">";
     }
 
-    @Override
```

Comment:
IMO previous version of hashcode+equals was more readable

also it not clear why to change the order :)

Reply:
I used an auto generated eclipse equals and hashcode. I could mimic the style of the original one indeed. The important thing for me is the fields that are included in the equality and hashcode testing to avoid the mem leak !